cmd/cosmos-sdk-cli/cmd/root.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "cosmos-sdk-cli",
+	Short: "Tools to develop on cosmos-sdk",
+	Run: func(cmd *cobra.Command, args []string) {
+		// Do Stuff Here
+	},
```

Comment:
I did think about adding a new command to create a new extension - keeper.go, handler.go, types.go, tx.go etc. I've some basic experience developing on cosmos-sdk and I think this would be beneficial from a user perspective.

Another command that would be beneficial would be to create a new Message that goes in types.go within extension folder. Because we have the basic functions like GetSigner, GetAccount, that needs to be added to every Message.

I will remove this empty function though.

Reply:
Cool, that makes sense 👍